src/dmd/mtype.h

```diff
@@ -119,9 +119,9 @@ typedef unsigned char MOD;
 
 // These tables are for implicit conversion of binary ops;
 // the indices are the type of operand one, followed by operand two.
-extern unsigned char impcnvResult[TMAX][TMAX];
-extern unsigned char impcnvType1[TMAX][TMAX];
-extern unsigned char impcnvType2[TMAX][TMAX];
+extern TY impcnvResult[TMAX][TMAX];
+extern TY impcnvType1[TMAX][TMAX];
+extern TY impcnvType2[TMAX][TMAX];
```

Comment:
@ibuclaw You mean these? Is there a way to specify that the enum should be 8 bits in c++98?

Reply:
See #9268 and #9191 for the last time @WalterBright changed an enum on the D side.